packages/babel-plugin-proposal-async-generator-functions/test/fixtures/async-generators/transform-class-keys/output.js

```diff
@@ -0,0 +1,18 @@
+function fn() {
+  return _fn.apply(this, arguments);
+}
+
+function _fn() {
+  _fn = babelHelpers.wrapAsyncGenerator(function* () {
+    class A {
+      [yield 1]() {}
+
+    }
+
+    class B extends A {
+      [await 1]() {}
```

Comment:
This test is (as expected) already producing the wrong output: this await here is invalid.

Reply:
oh, I thought I must get `SyntaxError: missing ) after argument list`.

Reply:
That's probably the error that we would get by executing this output file, but this wrong output is enough to show the bug!